test/system/helpers.bash

```diff
@@ -473,6 +483,15 @@ function skip_if_root_ubuntu {
     fi
 }
 
+######################
+#  skip_if_netavark  #  ...with an optional message
+######################
+function skip_if_netavark() {
+    if is_netavark; then
+        skip "${1:-not supported with netavark network backend}"
+    fi
+}
+
```

Comment:
This is not currently used. Could we just delete it, and assume that someone will add it if the need arises?

Reply:
sure.